api/models/Centres.js

```diff
@@ -153,7 +169,18 @@ const model = {
         }
         return centre[0];
       });
-  }
+  },
+
+  publishCentreUpdates: prebookings =>
+    Centres.find()
+      .populate('male_prebooking')
+      .populate('female_prebooking')
+      .populate('male_active_movements_in')
+      .populate('male_active_movements_out')
+      .populate('female_active_movements_in')
+      .populate('female_active_movements_out')
+      .then(centres => _.map(centres, centre => Centres.publishUpdate(centre.id, centre.toJSON())))
```

Comment:
can this not be:
`.map(centre => Centres.publishUpdate(centre.id, centre.toJSON()))`?